docs/pages/management/admin/labels.mdx

```diff
@@ -1,405 +1,323 @@
 ---
-title: Labels
+title: Add Labels to Resources
```

Comment:
The title and description are effectively identical now. Is that intended?

Reply:
Actually, I think the description could be a bit more descriptive.